Sources/Subprocess/Buffer.swift

```diff
@@ -88,44 +80,46 @@ extension AsyncBufferSequence.Buffer {
-        self.data.enumerateBytes { buffer, byteIndex, stop in
-            if _fastPath(backing == nil) {
-                // In practice, almost all `DispatchData` is contiguous
-                backing = .pointer(buffer)
-            } else {
-                // This DispatchData is not contiguous. We need to copy
-                // the bytes out
-                let contents = Array(buffer)
-                switch backing! {
-                case .pointer(let ptr):
-                    // Convert the ptr to array
-                    let existing = Array(ptr)
-                    backing = .array(existing + contents)
-                case .array(let array):
-                    backing = .array(array + contents)
+        @lifetime(borrow self)
+        borrowing get {
+            var backing: SpanBacking?
+            #if os(Windows)
+            self.data.withUnsafeBufferPointer {
+                backing = .pointer($0)
+            }
+            #else
+            self.data.enumerateBytes { buffer, byteIndex, stop in
+                if _fastPath(backing == nil) {
+                    // In practice, almost all `DispatchData` is contiguous
+                    backing = .pointer(buffer)
+                } else {
+                    // This DispatchData is not contiguous. We need to copy
+                    // the bytes out
+                    let contents = Array(buffer)
+                    switch backing! {
+                    case .pointer(let ptr):
+                        // Convert the ptr to array
+                        let existing = Array(ptr)
+                        backing = .array(existing + contents)
+                    case .array(let array):
+                        backing = .array(array + contents)
+                    }
                 }
             }
-        }
-        #endif
-        guard let backing = backing else {
-            let empty = UnsafeRawBufferPointer(start: nil, count: 0)
-            let span = RawSpan(_unsafeBytes: empty)
-            return _overrideLifetime(of: span, to: self)
-        }
-        switch backing {
-        case .pointer(let ptr):
-            let span = RawSpan(_unsafeElements: ptr)
-            return _overrideLifetime(of: span, to: self)
-        case .array(let array):
-            let ptr = array.withUnsafeBytes { $0 }
-            let span = RawSpan(_unsafeBytes: ptr)
-            return _overrideLifetime(of: span, to: self)
+            #endif
+            guard let backing = backing else {
+                let empty = UnsafeRawBufferPointer(start: nil, count: 0)
+                let span = RawSpan(_unsafeBytes: empty)
+                return _overrideLifetime(of: span, to: self)
+            }
+            switch backing {
+            case .pointer(let ptr):
+                let span = RawSpan(_unsafeElements: ptr)
+                return _overrideLifetime(of: span, to: self)
+            case .array(let array):
+                let span = array.span.bytes
```

Comment:
Swift 6.2 on Windows: error: value of type '[UInt8]' has no member 'span'

Reply:
Hmm that's strange... should be available on `6.2`. Let me check